code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -256,6 +263,58 @@
 			qdel(src)*/
 
 
+/obj/item/weapon/slimestabilizer
+	name = "slime stabilizer"
+	desc = "A potent chemical mix that will reduce the chance of a slime mutating."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle15"
+
+/obj/item/weapon/slimestabilizer/attack(mob/living/simple_animal/slime/M as mob, mob/user as mob)
+	if(!isslime(M))
+		user << "<span class='warning'> The stabilizer only works on baby slimes!</span>"
+		return ..()
+	if(M.stat)
+		user << "<span class='warning'> The slime is dead!</span>"
+		return..()
+	if(M.mutation_chance == 0)
+		user <<"<span class='warning'> The slime already has no chance of mutating!</span>"
+		return..()
+
+	user <<"<span class='notice'>You feed the slime the stabilizer. It is now less likely to mutate.</span>"
+	M.mutation_chance -= 15
+	if(M.mutation_chance < 0)
+		M.mutation_chance = 0
+	qdel(src)
+
+/obj/item/weapon/slimemutator
+	name = "slime mutator"
+	desc = "A potent chemical mix that will increase the chance of a slime mutating."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle3"
+
+/obj/item/weapon/slimemutator/attack(mob/living/simple_animal/slime/M as mob, mob/user as mob)
+	if(!isslime(M))
+		user << "<span class='warning'> The mutator only works on baby slimes!</span>"
+		return ..()
+	if(M.stat)
+		user << "<span class='warning'> The slime is dead!</span>"
+		return..()
+	if(M.mutator_used == 1)
```

Comment:
swap this for ```if(M.mutator_used)``` 

alternatively, make it a proper boolean with
```mutator_used = FALSE```
and then this check remaining ```if(M.mutator_used)